tests/test_spider.py

```diff
@@ -28,9 +28,11 @@
 )
 from scrapy.spiders.init import InitSpider
 from scrapy.utils.test import get_crawler
+from scrapy.exceptions import ScrapyDeprecationWarning
 from tests import get_testdata, tests_datadir
 
 
+
```

Comment:
This suggests you haven't used `pre-commit`, as described in https://docs.scrapy.org/en/latest/contributing.html#pre-commit